kedro-org/kedro-viz
# Adjust exp. tracking CSS to account for tall matplotlib images

Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Fixes #1144.

## Development notes

I've adjusted the CSS to allow for very tall images in experiment tracking. Before this PR we were limiting the height and now we aren't.

## QA notes

The run `2022-09-05T12.27.04.496Z` has a very tall image in there now. Check that run to see that the image isn't covering any of the other elements on the page like in the screenshot in the linked issue. I'll remove this test image before merging the PR.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1145"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
